apache-rat/src/site/apt/index.apt.vm

```diff
@@ -75,17 +75,17 @@ java -jar apache-rat/target/apache-rat-${project.version}.jar
 
 
 +------------------------------------------+
-usage: java -jar apache-rat/target/apache-rat-${project.version}.jar
- [options] [DIR|TARBALL]
+usage: java -jar apache-rat/target/apache-rat-CURRENT-VERSION.jar
```

Comment:
Where does the CURRENT-VERSION come from as project.version is standard Maven file filtering? Maybe I do not understand the mechanism properly (sorry for asking)